console/module/device/src/main/java/org/eclipse/kapua/app/console/module/device/client/device/DeviceDeleteDialog.java

```diff
@@ -58,9 +59,9 @@ public void submit() {
                 new AsyncCallback<Void>() {
 
                     public void onFailure(Throwable caught) {
+                        FailureHandler.handle(caught);
```

Comment:
Handling of the error is managed by the `EntityCRUDToolbar` so is enough to set `exitStatus` and `exitMessage`. 

This call to the `FailureHandler` can be removed